5.2.cup_of_join.py

```diff
@@ -0,0 +1,19 @@
+# The function accepts an unlimited number of lists and an option for an additional parameter,
+# and concatenates them into one list and the additional parameter between them
+def join(*lists, sep='-'):
+    if not lists:  # If no lists were received at all
+        return None
+    lst = [sep]
```

Comment:
res and lst are not good variable names, Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Reply:
I fixed it.